mysql-test/suite/encryption/t/innodb_encryption_discard_import.test

```diff
@@ -1,9 +1,9 @@
---skip
 --source include/not_valgrind.inc
 
-call mtr.add_suppression("\\[Warning\\] InnoDB: Tablespace for table `[^`]+`.`[^`]+` is set as discarded\\.");
-call mtr.add_suppression("\\[Warning\\] InnoDB: Page [[:digit:]]+ at offset [[:digit:]]+ looks corrupted in file");
-call mtr.add_suppression("\\[ERROR\\] InnoDB: os_file_read\\(\\) failed");
+call mtr.add_suppression("\\[InnoDB\\] Tablespace id .* in file t[1-8]\.ibd is encrypted but keyring or used key_id [4-5] is not available\. Can't continue reading table\. Please provide the correct keyring\.");
```

Comment:
Same question as for the other test, is this range enough?

Reply:
should be  :)